src/utilities/generateMeta.ts

```diff
@@ -47,3 +47,32 @@ export const generateMeta = async (args: {
     title,
   }
 }
+
+export const generateMetaPost = async (args: {
+  customTitle?: string
+  center: string
+  doc: Partial<Post>
+}): Promise<Metadata> => {
+  const { customTitle, center, doc } = args || {}
+
+  const ogImage = getImageURL(doc?.featuredImage)
+
+  const title = customTitle ? customTitle : `${doc?.title} ' | '${center}`
+
+  return {
+    description: doc?.description,
+    openGraph: mergeOpenGraph({
+      description: doc?.description || '',
+      images: ogImage
+        ? [
+            {
+              url: ogImage,
+            },
+          ]
+        : undefined,
+      title,
+      url: Array.isArray(doc?.slug) ? doc?.slug.join('/') : '/',
```

Comment:
This might be a hold-over from the website template that doesn't work for our site-map.

Reply:
This is from the template. I will update it.